src/use_case/edit_friends/EditFriendsInteractor.java

```diff
@@ -1,4 +1,22 @@
 package use_case.edit_friends;
 
-public class EditFriendsInteractor {
```

Comment:
functions not implemented yet first we need and output layer and output data-to be discussed